include/TrustWalletCore/TWBlockchain.h

```diff
@@ -39,6 +39,7 @@ enum TWBlockchain {
     TWBlockchainWaves = 25,
     TWBlockchainAeternity = 26,
     TWBlockchainNebulas = 27,
+    TWBlockchainAuxilium = 28945486,
```

Comment:
@viktoriia-boichuk this list is clearly not of Ethereum chainIDs: please check if it's possible to just use sequential number here, 28